geometry_msgs/Pose approach_pose
---
bool approach_reply